libs/core/langchain_core/tools/convert.py

```diff
@@ -66,6 +70,7 @@ def tool(
     name_or_callable: Optional[Union[str, Callable]] = None,
     runnable: Optional[Runnable] = None,
     *args: Any,
+    description: Optional[str] = None,
```

Comment:
we'll need to document the new option so it appears in the API reference.


Also to confirm description isn't captured in the args schema at all?

Reply:
it is, i added a test for that